src/preferenceCenter/preferenceCenter.blocks.js

```diff
@@ -0,0 +1,176 @@
+export default class PreferenceCenterBlocks {
+  blockManager;
+
+  constructor(editor) {
+    this.blockManager = editor.BlockManager;
+  }
+
+  addPreferenceCenterBlock() {
+    // add preference center category-list block
+    this.blockManager.add('MyCategories', {
```

Comment:
I guess this should be renamed to something more specific. E.g. Preference Center?

Reply:
I can change block's name while we add using blockManager.add() but the label of that is "My Category" and I gave same name from the legacy builder. Preference Ceneter is category. Like we have "section" , "block" for preference center pages we will have "Prefernce Center" and when you open it under that you will find Preference ceneter related blocks MyCategory, My Segement, Save Preference etc. If we want to change, please let me know what that label would be.